package-lock.json

```diff
@@ -0,0 +1,5496 @@
+{
+  "name": "konflux-ui",
```

Comment:
Why is there package-lock as well as yarn-lock?

Reply:
removed package-lock file